### PR TITLE
Add path_alias explicitly for gcp-compute-persistent-disk-csi-driver jobs

### PR DESCRIPTION
**Pull request by dims**
fix for https://github.com/kubernetes/test-infra/issues/30898

### CODE REVIEW

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Review by upodroid — APPROVED** [review]
LGTM

**Comment by sunnylovestiramisu** [comment]
The tests are still failing but for a different reason, how did you decide the correct path for the test setup?

**Comment by BenTheElder** [comment]
This config is equivalent to the default behavior in bootstrap.py of aliasing kubernetes repos to sigs.k8s.io/$repo or k8s.io/$repo. pod utilities doesn't encode as much kubernetes specific behavior and if you want the same checkout location you have to configure the path_alias, otherwise you get $GOPATH/src/go/$repo_url (repo_url=github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver)

https://github.com/kubernetes/test-infra/issues/30898 has more follow up besides this PR

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ presubmits:
     decorate: true
     decoration_config:
       timeout: 30m
+    path_alias: sigs.k8s.io/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml:11]: oh, this is a goofy path alias? sigs is repeated.

filing follow-up

**Comment by BenTheElder** [comment]
@sunnylovestiramisu I think I understand what you're getting at now 😅 , filed https://github.com/kubernetes/test-infra/pull/30904

**Pull request merged**